antora-ui-camel/src/partials/footer-scripts.hbs

```diff
@@ -1,5 +1,6 @@
 <script src="{{uiRootPath}}{{asset 'js/site.js'}}"></script>
 <script async src="{{uiRootPath}}{{asset 'js/vendor/highlight.js'}}"></script>
 <script async src="{{uiRootPath}}{{asset 'js/vendor/svg4everybody.js'}}"></script>
+<script async src="https://cdn.jsdelivr.net/npm/algoliasearch@4/dist/algoliasearch-lite.umd.js"></script>
```

Comment:
The content security policy won't load this on the production site. We need to put this script in the [`vendor`](https://github.com/apache/camel-website/tree/master/antora-ui-camel/src/js/vendor) directory and load it from there. Just like the two other scripts we have here.

Reply:
@zregvart I tried to solve this by 
first
>yarn add algoliasearch in antora-ui-camel 

then
>added a new algolia_search.js file into vendor folder which has
;(function () {
  var algoliasearch = require('algoliasearch')
  algoliassearch()
})()

finally
>add that script into to footer scrips as you mentioned

then I`m having these errors, any thoughts? 

![Screenshot from 2020-03-03 08-06-46](https://user-images.githubusercontent.com/25351304/75737567-14f59280-5d26-11ea-993e-2601c50495a6.png)
![Screenshot from 2020-03-03 08-05-37](https://user-images.githubusercontent.com/25351304/75737584-1de66400-5d26-11ea-9920-bb20ecc093b3.png)

Reply:
You need to copy the file to the vendor directory, the Antora UI build is not using the dependencies declared in package.json. The idea of vendoring dependencies is that we copy the dependency that we use into our own source tree and not pull it in through any dependency management (such as npm or yarn).

Reply:
It's much simpler to use the ui builder to pull in and browserify the sources. You do this with code like Nayananga suggested:
```
;(function () {
var algoliasearch = require('algoliasearch')
return algoliasearch()
})()
````
listing the `algoliasearch` in your package.json for the ui project, but naming the file `algoliasearch.bundle.js`.  The 'bundle' in the filename tells the builder to convert the require into a sort of 'include'.

Reply:
@djencks Thanks for the suggestion, I was able to go half way through in that way by renaming the script, but then it gave me an error
> window.algoliasearch is not a function.

So I had to copy 06-search.js into the algoliasearch.bundle.js (which I`ve commited just now) now its work fine :)